sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-generation.service.ts

```diff
@@ -48,14 +48,18 @@ export class ECertGenerationService {
    * - `offeringIntensity`: offering intensity to retrieve the disbursements.
    * - `applicationId`: restricts the query to a specific application.
    * - `checkDisbursementMinDate`: check only for disbursements that are close
+   * - `allowNonCompleted`: only select completed applications or allow any status.
    * to the date to be disbursed and can already be part of an e-Cert.
```

Comment:
same here. You need to swap lines 51 and 52.

Reply:
fixed